### ISSUE TITLE
Error after updating quicklisp/ultralisp

### ISSUE THREAD

**Issue by simendsjo** [alex-gutev/cl-form-types#20]
sbcl 2.3.10 on Windows 11

```
[package cl-form-types.walker].............;
; caught ERROR:
;   READ error during COMPILE-FILE:
;
;     Symbol "TRULY-DYNAMIC-EXTENT" not found in the SB-INT package.
;
;       Line: 456, Column: 69, File-Position: 13897
;
;       Stream: #<SB-INT:FORM-TRACKING-STREAM for "file C:\\Users\\simend\\quicklisp\\dists\\ultralisp\\software\\alex-gutev-cl-form-types-20230614064654\\src\\walker.lisp" {100CA31213}>
```

**Comment by simendsjo** [comment]
Works on sbcl 2.3.9.
I see the following changelog entry in 2.3.9
```
enhancement: stack allocation via DYNAMIC-EXTENT now applies to all values that a variable can take on (for example via SETQ), not just the initial binding. This permits for example building complex or recursive structures on the stack more easily via iteration. See the updated manual entry for more details. 
```

Maybe this from 2.3.10 is related?
```
enhancement: The compiler now allows stack allocating vectors of any size on all safety levels, not just those which it can prove are of sub-page sizes. It can do this because it now inserts code to check for stack overflow explicitly on higher safety levels. 
```

**Comment by alex-gutev** [comment]
Should be fixed in the latest commit d7d7b0e.
**NOTE:** You'll need the update [cl-environments](https://github.com/alex-gutev/cl-environments) to the latest commit.
